dll/win32/ucrtbase/stubs.c

```diff
@@ -0,0 +1,83 @@
+
```

Comment:
missing file copyright header

Reply:
It's a temporary stubs file, it will go away.